cluster/juju/layers/kubernetes-worker/templates/ingress-replication-controller.yaml

```diff
@@ -45,3 +45,4 @@ spec:
         args:
         - /nginx-ingress-controller
         - --default-backend-service=$(POD_NAMESPACE)/default-http-backend
+        - --nginx-configmap=$(POD_NAMESPACE)/nginx-load-balancer-conf
```

Comment:
Thanks for the contribution!  I see you are adding a configmap flag here. How are you going to get the Nginx configuration to the worker?